NuRadioReco/modules/RNO_G/dataProviderRNOG.py

```diff
@@ -21,6 +50,17 @@ def __init__(self):
 
 
     def begin(self, files, reader_kwargs={}, det=None):
```

Comment:
if det is required, why not use `begin(self, files, det, **reader_kwargs)`? (or `reader_kwargs={}` as now)? Not that it makes a big difference...

Reply:
done